menu_technical_info/static/src/js/menu_technical_info.js

```diff
@@ -0,0 +1,33 @@
+odoo.define('menu_technical_info.Menu', function (require) {
```

Comment:
Could you review the eslint output?
```bash
eslint menu_technical_info.js
menu_technical_info.js
  10:23  error  Avoid using Function.prototype.apply, instead use Reflect.apply  prefer-reflect
  10:23  error  use the spread operator instead of the '.apply()'                prefer-spread
  10:47  error  Use the rest parameters instead of 'arguments'                   prefer-rest-params
  11:26  error  Gratuitous parentheses around expression                         no-extra-parens
  11:70  error  Unexpected use of undefined                                      no-undefined
  21:13  error  Expected { after 'if' condition                                  curly
  23:13  error  Expected { after 'if' condition                                  curly
```

(No blocking)